syntax = "proto3";

package RegisterProtocol;

message UserInfo
{
    string account = 1;
    string password = 2;
    string name = 3;
}

message Verification
{
    string verification_code = 1;
}